### ISSUE TITLE
Possible bug?

### ISSUE THREAD

**Issue by bernaferrari** [beenotung/best-effort-json-parser#2]
I'm not sure if I'm using your library as intended, but I'm getting a lot of errors, like `no parser registered` on:

```js
var bestEffortJsonParser = require("best-effort-json-parser")

const a = `{
a: "abc"`

let data = bestEffortJsonParser.parse(a)
console.log(data)
```

```js
var bestEffortJsonParser = require("best-effort-json-parser")

const a = `{
a: "abc",
b: "def"
}`

let data = bestEffortJsonParser.parse(a)
console.log(data)
```

**Comment by beenotung** [comment]
In a proper JSON string, the object key should be wrapped with double quotation.

For example:
```json
{
  "a": "abc",
  "b": "def"
}
```

Maybe this library can do better effort to handle this kind of invalid but understandable input as well

**Comment by bernaferrari** [comment]
yeah, some error handling would be better.. I ended up just using a different library

**Comment by beenotung** [comment]
The update is released to npm as `best-effort-json-parser@1.1.0` :rocket: 

Feel free to re-open the issue if it doesn't address your case
